test/controllers/api/v1/api_controller_test.rb

```diff
@@ -165,6 +165,64 @@ def setup
     assert_equal 0, validator_with_all_data['autonomous_system_number']
   end
 
+  # 
+  # Pagination
+  #
+  test 'GET api_v1_validators with token, limit and page passed in returns limited data' do
+    create_list(:validator, 10, :with_score)
+    limit = 5
+    page = 3
+
+    get api_v1_validators_url(network: 'testnet', limit: limit, page: page),
+        headers: { 'Token' => @user.api_token }
+
+    assert_response 200
+    json = response_to_json(@response.body)
+
+    # There are 11 validators, so last page with limit 5 should return 1 record.
```

Comment:
if there are some old validators from fixtures, maybe it's a good idea to delete them in setup to clarify this specific tests